pkgs/by-name/hy/hydrus/package.nix

```diff
@@ -0,0 +1,172 @@
+{
+  lib,
+  stdenv,
+  python3Packages,
+  fetchFromGitHub,
+  libsForQt5,
```

Comment:
in [`4b01a04` (#387547)](https://github.com/NixOS/nixpkgs/pull/387547/commits/4b01a042c983f43e668a730ef2c27f69a4e0f532) it seems like you downgraded hydrus from qt6 to qt5? hydrus does not support qt5 officially